tests/tests_pytorch/callbacks/test_sanity_serving.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+import torch
+
+from pytorch_lightning import seed_everything, Trainer
+from pytorch_lightning.callbacks.sanity_serving import SanityServing, ServableModule
+from pytorch_lightning.demos.boring_classes import BoringModel
+
+
+class ServableBoringModel(BoringModel, ServableModule):
+    def configure_payload(self) -> ...:
+        return {"body": {"x": list(range(32))}}
+
+    def configure_inputs_outputs(self):
```

Comment:
I would suggest a name like `configure_serialization` instead. Inputs/outputs is a very broad term

Reply:
Done !